webapp/src/components/DatasetWarnings/DatasetDistribution.tsx

```diff
@@ -9,13 +9,13 @@ import { DatasetWarningGroup } from "types/api";
 const DESCRIPTION = {
   "General Warnings": (
     <Description
-      text="See class distributions differences."
+      text="Investigate issues with class size, class imbalance and dataset shift."
```

Comment:
I think I'd add the oxford comma, but that's a stylistic choice! We should probably be consistent as we change things, though.

```suggestion
      text="Investigate issues with class size, class imbalance, and dataset shift."
```